Library/Formula/node.rb

```diff
@@ -21,7 +22,7 @@ class Node < Formula
 
   depends_on :python => :build if MacOS.version <= :snow_leopard
   depends_on "pkg-config" => :build
-  depends_on "openssl" => :optional
```

Comment:
I tried leaving it in place and it complained that `--without-ssl2` is not a supported parameter.